Readme.md

```diff
@@ -19,25 +21,37 @@ Allure.XUnit supports .NET Core 2.0 and later.
 * AllureSubSuite
 * AllureLink
 * AllureEpic
-* AllureXunitTheory - attribute for display parametrized tests. Use ```InlineData```, ```MemberData```, ```ClassData```, XUnit attributes for pass parameters.
-* AllureAddAttachment - attribute for attach files to tests. AllureAddAttachment take as paramet file path or Type of object witch implement   IAllureGetAttachmentPath interface for calculate attachment file path.
-```
-IAllureGetAttachmentPath
-{
-       string GetAttachmentPath(); //calculate attachment file path in runtime
-}
-```
- All methods have to be tagged by attribute AllureXunit inteasd of Fact, or AllureXunitTheory instead of Theory for display in allure report. Other attributes are optional.
-
- ## Running
- For start use: dotnet test.
- 
- allure-results directory with result appears after running tests in target dirrectory 
-
- Execute command ```allure serve ./ ``` in allure-results directory to local viewing tests.
+* AllureLabel
+* AllureXunitTheory - attribute for display parametrized tests. Use ```InlineData```, ```MemberData```, ```ClassData```,
+  XUnit attributes for pass parameters.
+
+All methods have to be tagged by attribute AllureXunit instead of Fact, or AllureXunitTheory instead of Theory for
+display in allure report. Other attributes are optional.
+
+## Steps
+There are two ways to describe steps:
+1. Use [`Steps`](src/Allure.XUnit/Steps.cs) class for functional or imperative approach.
+2. Use `AllureStepAttribute`, `AllureBeforeAttribute`, `AllureAfterAttribute` for declarative approach.
+
+See [Examples](src/Examples/ExampleSteps.cs).
+
+## Attachments
+Use [`AllureAttachments`](src/Allure.XUnit/AllureAttachments.cs) class with it's methods.
+
+## Running
+
+Just run `dotnet test`.
+
+`allure-results` directory with result appears after running tests in target directory.
+
+Execute command ```allure serve ./ ``` in allure-results directory to local viewing tests. Also you can browse result
+using similar [docker-compose file](./src/Examples/docker-compose.yaml).
 
 ## Examples
+
 [Examples](src/Examples)
 
- ## Author
- [Shumakov Ivan](https://github.com/IvanWR1995)
+## Contributors
+
+* [Shumakov Ivan](https://github.com/IvanWR1995)
+* [Artamonov Ruslan](https://github.com/rus-art)
```

Comment:
I don't agree with this change. Let's leave the
paragraph "author" how it was and let's add the paragraph "contributors".

Reply:
Ok)